api/src/pcapi/routes/backoffice_v3/offers/forms.py

```diff
@@ -0,0 +1,256 @@
+    )
+    order = wtforms.HiddenField(
+        "order", default="asc", validators=(wtforms.validators.Optional(), wtforms.validators.AnyOf(("asc", "desc")))
+    )
+    limit = fields.PCSelectField(
+        "Nombre maximum de résultats",
+        choices=((100, "100"), (500, "500"), (1000, "1000"), (3000, "3000")),
+        default="100",
+        coerce=int,
+        validators=(wtforms.validators.Optional(),),
+    )
+
+    def is_empty(self) -> bool:
+        # needed to clean multiple inheritance
+        return True
+
+
+class OfferAdvancedSearchSubForm(utils.PCForm):
+    class Meta:
+        csrf = False
+
+    json_data = json.dumps(
+        {
+            "display_configuration": form_field_configuration,
+            "all_available_fields": [
+                "category",
+                "criteria",
+                "date",
+                "department",
+                "integer",
+                "offerer",
+                "string",
+                "venue",
+                "validation",
+            ],
+            "sub_rule_type_field_name": "search_field",
+            "operator_field_name": "operator",
+        }
+    )
+
+    def __init__(self, *args: list, **kwargs: dict):
+        super().__init__(*args, **kwargs)
+        autocomplete.prefill_criteria_choices(self.criteria)
+        autocomplete.prefill_offerers_choices(self.offerer)
+        autocomplete.prefill_venues_choices(self.venue)
+
+    search_field = fields.PCSelectWithPlaceholderValueField(
+        "champ de recherche",
+        choices=utils.choices_from_enum(SearchAttributes),
+        validators=[
+            wtforms.validators.Optional(""),
+        ],
+    )
+    operator = fields.PCSelectWithPlaceholderValueField(
+        "Opérateur",
+        choices=utils.choices_from_enum(SearchOperators, formatter=filters.format_offer_validation_operator),
+        validators=[
+            wtforms.validators.Optional(""),
+        ],
+    )
+    category = fields.PCSelectMultipleField(
+        "Catégories", choices=utils.choices_from_enum(categories.CategoryIdLabelEnum)
+    )
+    criteria = fields.PCTomSelectField(
+        "Tags", multiple=True, choices=[], validate_choice=False, endpoint="backoffice_v3_web.autocomplete_criteria"
+    )
+    department = fields.PCSelectMultipleField("Départements", choices=dict(constants.area_choices))
+    integer = fields.PCOptIntegerField(
+        "Valeur numérique",
+        validators=[
+            wtforms.validators.Optional(""),
+            wtforms.validators.NumberRange(min=0, max=(2**63) - 1, message="Doit être inférieur à %(max)d"),
+        ],
+    )
+    offerer = fields.PCTomSelectField(
+        "Structures",
+        multiple=True,
+        choices=[],
+        validate_choice=False,
+        endpoint="backoffice_v3_web.autocomplete_offerers",
+    )
+    string = fields.PCOptStringField(
+        "Chaîne",
```

Comment:
```suggestion
        "Chaîne",
```
Excusez-moi, vous travaillez dans l'informatique ? 😛 
Peut-être un autre mot ("Texte", etc) ?

Reply:
Vous êtes désobligeant jeune homme.
(oui Texte passe mieux, je le prend)